static/templates/admin/plugins/openedx-discussion.tpl

```diff
@@ -4,42 +4,64 @@
 		<div class="col-sm-10 col-xs-12">
 			<p class="lead">
 				Configure these settings to share session between openEdx and nodebb instance.
-
-				JWT Cookie Name: Name of the encoded jwt cookie that will be sent with requests to nodebb.
-				Secret: `key` for "key:value" pair of secret used for jwt encoding and decoding.
-				Redirect Login Url: `login-url` for where to direct an user when he tries to login.
-				Redirect Registration Url: `registration-url` for where to direct an user when he tries to register.
-				Logout Url: `logout-url` is the openedx logout url which will be used to clear session when user logs out from nodebb.
 			</p>
 
-			<div class="form-group">
-				<label for="jwtCookieNamee">JWT cookie name</label>
-				<input type="text" id="jwtCookieName" name="jwtCookieName" title="JWT Cookie Name" class="form-control" placeholder="Token">
+			<div class="row panel panel-body">
+				<div class="form-group">
+					<label for="jwtCookieName">JWT cookie name</label>
+					<input type="text" id="jwtCookieName" name="jwtCookieName" title="JWT Cookie Name"
+						class="form-control" placeholder="Token">
+				</div>
+				<p>JWT Cookie Name: Name of the encoded jwt cookie that will be sent with requests to nodebb.</p>
+
 			</div>
 
-			<div class="form-group">
-				<label for="secret">Secret</label>
-				<input type="text" id="secret" name="secret" title="secret" class="secret" placeholder="its a secret">
+			<div class="row panel panel-body">
+				<div class="form-group">
+					<label for="secret">Secret</label>
+					<input type="text" id="secret" name="secret" title="secret" class="form-control"
+						placeholder="its a secret">
+					<p>Secret: `key` for "key:value" pair of secret used for jwt encoding and decoding.</p>
+				</div>
 			</div>
 
-			<div class="form-group">
-				<label for="loginURL">Login URL</label>
-				<input type="text" id="loginURL" name="loginURL" title="loginURL" class="loginURL" placeholder="https://example.com/login">
+			<div class="row panel panel-body">
+				<div class="form-group">
+					<label for="loginURL">Login URL</label>
+					<input type="text" id="loginURL" name="loginURL" title="loginURL" class="form-control"
+						placeholder="https://example.com/login">
```

Comment:
```suggestion
						placeholder="https://example.com/login" required>
```

Reply:
This thing is not actually required. This is just there is we want a single login page for both edx and nodebb. but is someone wants to run nodebb as seperate instance as well where users can register and login without having a link to openedx, then these fields can be left empty.

Reply:
OK